lib/class-wp-rest-block-types-controller.php

```diff
@@ -239,21 +239,23 @@ public function prepare_item_for_response( $block_type, $request ) {
 
 		$schema       = $this->get_item_schema();
 		$extra_fields = array(
-			'name'          => 'name',
-			'title'         => 'title',
-			'description'   => 'description',
-			'icon'          => 'icon',
-			'category'      => 'category',
-			'keywords'      => 'keywords',
-			'parent'        => 'parent',
-			'supports'      => 'supports',
-			'styles'        => 'styles',
-			'textdomain'    => 'textdomain',
-			'example'       => 'example',
-			'editor_script' => 'editor_script',
-			'script'        => 'script',
-			'editor_style'  => 'editor_style',
-			'style'         => 'style',
+			'name'             => 'name',
+			'title'            => 'title',
+			'description'      => 'description',
+			'icon'             => 'icon',
+			'category'         => 'category',
+			'keywords'         => 'keywords',
+			'parent'           => 'parent',
+			'supports'         => 'supports',
+			'styles'           => 'styles',
+			'textdomain'       => 'textdomain',
+			'example'          => 'example',
+			'editor_script'    => 'editor_script',
+			'script'           => 'script',
+			'editor_style'     => 'editor_style',
+			'style'            => 'style',
+			'provides_context' => 'providesContext',
+			'use_context'      => 'context',
```

Comment:
I still believe having this named context here is really confusing for the REST API. Context is a keyword already in use in the API. Can't it be useContext here?

Reply:
Isn't it `use_context` in the API response and `context` on `WP_Block_Type`. I might be missing something.

Reply:
I saw that, having `context` is the response is confusing.

Reply:
```suggestion
			'use_context'      => 'use_context',
```